src/test/groovy/nebula/plugin/info/InfoPluginIntegrationTestKitSpec.groovy

```diff
@@ -38,7 +38,6 @@ class InfoPluginIntegrationTestKitSpec extends IntegrationTestKitSpec {
              java {
                 toolchain {
                     languageVersion = JavaLanguageVersion.of(17)
-                    vendor = JvmVendorSpec.ADOPTIUM
```

Comment:
This is just to avoid broken builds for folks that don't use adoptium so they can run the tests locally with any vendor as long as they have java 17 installed